src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -96,22 +96,23 @@ export default class AgreementType extends Vue {
   private showAgreementTypeForm = false
 
   mounted (): void {
-    this.setIncorporationAgreementStepData({
-      valid: !!this.agreementType,
-      agreementType: this.agreementType
-    })
-
+    this.setIncorporationAgreementData()
```

Comment:
Is this needed, since the watch on line 141 has "immediate=true" (meaning it fires when the component is created I think) ?

Reply:
Probably not. I will check

Reply:
Fixed